src/diff.jl

```diff
@@ -171,7 +171,7 @@ function expand_derivatives(O::Symbolic, simplify=false; occurances=nothing)
             x = +((!_iszero(c) ? vcat(c, exprs) : exprs)...)
             return simplify ? SymbolicUtils.simplify(x) : x
         end
-    elseif isa(operation(O), Integral )
+    elseif istree(O) && isa(operation(O), Integral )
```

Comment:
Could you be more careful about whitespace?

Reply:
I m sorry, I fixed that.